site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -80,6 +80,7 @@ The GKE cluster created from the example configuration will contain 3 Node Pools
 
 Configurable parameters:
 
+{{% feature expiryVersion="0.14.0" %}}
 - project - your Google Cloud Project ID (required)
 - name - the name of the GKE cluster (default is "agones-terraform-example")
 - agones_version - the version of agones to install (an empty string, which is the default, is the latest version from the [Helm repository](https://agones.dev/chart/stable))
```

Comment:
You could just put this line for n1-standard-4 in the feature expiry and replace it with the new note.

Reply:
I thought that doing it for a single line in a list breaks rendering for the whole list (but I could be wrong). Since the release candidate is tomorrow, I'm leaning towards merging this since it'll be removed soon in any case. WDYT?

Reply:
That's true for the table, but I am not sure about a list. You can locally verify using `make site-server`.
I am fine with either. You already have my approval.

Reply:
I did a local test and you are correct - you can just wrap the one list item with the feature expiry.

Reply:
Since this will be cleaned up in a week and I'd like to get this in before the release candidate gets cut, I'm going to merge as-is.

Reply:
Looking more closely, while it works, it changes the spacing of the list elements, which I find off-putting. So I think that doing it around the entire list is preferable.